src/rpc/budget.cpp

```diff
@@ -48,11 +53,110 @@ void budgetToJSON(CBudgetProposal* pbudgetProposal, UniValue& bObj)
     bObj.push_back(Pair("fValid", pbudgetProposal->fValid));
 }
 
-UniValue preparebudget(const UniValue& params, bool fHelp)
-{
-    int nBlockMin = 0;
+std::string validateURL(std::string strURL) {
```

Comment:
Even if we don't have to check for URLs in any place other than here, it could make sense to enable broader access.
Maybe move this to `utilstrencodings.h/.cpp` or `util.h/.cpp`?

Reply:
I didn't want to go so far as to go that far abstracted initially; but given the
> I like the proposed changes

I will  look for a better home for the URL routine

Reply:
will be addressed in #950